### PR TITLE
Update package.json with new dependencies

### PR DESCRIPTION
**Pull request by ankurkaushal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,16 +7,17 @@
...
-    "highlight.js": "~8.4.0",
+    "convert-source-map": "^1.1.1",
+    "express": "^5.0.0-alpha.2",
+    "highlight.js": "^8.9.1",
     "jade": "^1.11.0",
-    "less": "~1.7.4",
+    "jquery": "^2.1.4",
+    "less": "^2.5.3",
     "less-file": "0.0.9",
-    "promise": "7.0.4",
```
Comment (ForbesLindesay) [inline package.json]: I'm keeping `promise` bound to a specific version because it makes it easier to ensure I run a release after each update.

Comment (ankurkaushal) [inline package.json]: I see, so can the PR be merged or do I have to make another commit to correct that?
